syz-manager/manager.go

```diff
@@ -576,19 +577,34 @@ func (mgr *Manager) isSuppressed(crash *Crash) bool {
 	return false
 }
 
+func (mgr *Manager) emailCrash(crash *Crash) {
+	if mgr.cfg.Email_addr != "" {
+		Logf(0, "sending email to %v", mgr.cfg.Email_addr)
+		rpt := fmt.Sprintf("%s", crash.Report.Report)
+		cmdstr := "mailx -s " + "\"Syzkaller: " + crash.Title + "\" " + mgr.cfg.Email_addr + " <<< \"" + rpt + "\""
+		cmd := exec.Command("bash", "-c", cmdstr)
```

Comment:
Let's not shell out to do simple things, Go is perfectly capable of communicating with stdin:
```
	cmd := exec.Command("mailx", "-s", "syzkaller: " + crash.Title, mgr.cfg.Hub_Addr)
	cmd.Stdin = bytes.NewReader(crash.Report.Report)
	if _, err := osutil.Run(10 * time.Minute, cmd); err != nil {
		Logf(0, "failed to send email: %v", err)
	}
```